CONTRIBUTING.md

```diff
@@ -113,6 +113,11 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+-  If your PR includes changes to files under the ``doc/`` or
```

Comment:
Thanks, @jnothman!

Sorry, I think I misunderstood the comments in #7923 about what the markers did. Is something like this more appropriate?


A Note about CI: 
- Travis is used for testing on Linux platforms
- Appveyor is used for testing on Windows platforms
- CircleCI is used to build the docs for viewing

Please note that if one of the following markers appear in the latest commit message, the following actions are taken.

Commit Message Marker | Action Taken by CI
-|-
[ci skip] | CI is skipped completely
[doc skip] | Docs are not built 
[doc quick] | Docs built, but exclude example gallery plots
[doc build] | Docs built, including example gallery plots